modules/admin-top-bar/module.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Elementor\Modules\AdminTopBar;
+
+use Elementor\Core\Base\App as BaseApp;
+use Elementor\Core\Experiments\Manager;
+use Elementor\Plugin;
+use Elementor\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Module extends BaseApp {
+
+	/**
+	 * @return bool
+	 */
+	public static function is_active() {
+		return is_admin();
+	}
+
+	public static function get_experimental_data() {
+		return [
+			'name' => 'admin-top-bar',
+			'title' => __( 'Admin Top Bar', 'elementor' ),
+			'description' => __( 'Adds a top bar to elementors pages in admin area.', 'elementor' ),
+			'release_status' => Manager::RELEASE_STATUS_ALPHA,
+			'default' => Manager::STATE_INACTIVE,
+		];
+	}
+
+	/**
+	 * @return string
+	 */
+	public function get_name() {
+		return 'admin-top-bar';
+	}
+
+	private function render_admin_top_bar() {
+		?>
+		<div id="e-admin-top-bar">
+		</div>
+		<?php
+	}
+	protected function get_init_settings() {
+		return [
+			'is_administrator' => current_user_can( 'manage_options' ),
+		];
+	}
+
+	/**
+	 * Enqueue admin scripts
+	 */
+	private function enqueue_scripts() {
+		$direction_suffix = is_rtl() ? '-rtl' : '';
+
+		wp_register_style( 'elementor-admin-top-bar', $this->get_css_assets_url( 'admin-top-bar' . $direction_suffix ), [], ELEMENTOR_VERSION );
```

Comment:
`get_css_assets_url` is already handles the `rtl` asspect

Reply:
Done.